### PR TITLE
AboutUs page: make the list numbering continuous #6588

### LINKED ISSUE #6588: AboutUs page: make the list numbering continuous
See screenshot below. Instead of restarting the numbering from 1, make it continue from the previous list.

image

### PR DESCRIPTION
**Pull request by karthikaacharya**
PR Description: Fixes #6588 (Ready for review).

I have added the CSS counters in teammates.css in which it automatically increments the "li" count.

### CODE REVIEW

**Comment by AnaghHegde** [comment]
@karthikaacharya The contribution guide says like this : 

>PR name: copy-and-paste the relevant issue name and include the issue number as well, e.g. Remove unnecessary System.out.printlns from Java files #3942.

Just copy the issue name and use it as a PR title.

**Comment by karthikaacharya** [comment]
Sir, during the second edit I did the same thing as you said, issue name is "AboutUs page: make the list numbering continuous #6588" then also it gave same message.

**Comment by damithc** [comment]
@samsontmr can you check the bot response after the first edit? Seems the PR title was correct but the bot responded otherwise.

**Comment by samsontmr** [comment]
@damithc Strange, I am unable to reproduce this behavior on my testing repo even after adding a space at the end of the title.

**Comment by damithc** [comment]
Let me correct the title and see what happens.

**Comment by samsontmr** [comment]
I suspect the first comment was sent when the PR was opened, and the second when @karthikaacharya edited the PR description.

File: src/main/webapp/stylesheets/teammates.css [hunk 1]
```diff
@@ -105,6 +117,8 @@ h2 {
     width: 1000px;
 }
 
+
+
```
Comment (samsontmr) [inline src/main/webapp/stylesheets/teammates.css]: Remove these two new lines.

File: .settings/org.eclipse.jdt.core.prefs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
```
Comment (samsontmr) [inline .settings/org.eclipse.jdt.core.prefs]: This doesn't seem like something you should be pushing...

**Comment by damithc** [comment]
> I suspect the first comment was sent when the PR was opened, and the second when @karthikaacharya  edited the PR description.

The bot comment can be more detailed about what is the problem it is referring to. e.g. whether the problem is in the title or the description

File: src/main/webapp/stylesheets/teammates.css [hunk 3]
```diff
@@ -25,6 +25,18 @@ time, mark, audio, video {
...
 
+#firstList {
+   counter-reset:contributors;
+}
+
+li{
+   list-style-type:none;
+   counter-increment: contributors;
+}
+li:before {
```
Comment (samsontmr) [inline src/main/webapp/stylesheets/teammates.css]: Separate these two blocks with a newline

File: src/main/webapp/stylesheets/teammates.css [hunk 4]
```diff
@@ -25,6 +25,18 @@ time, mark, audio, video {
     vertical-align: baseline;
 }
 
+#firstList {
+   counter-reset:contributors;
+}
+
+li{
```
Comment (samsontmr) [inline src/main/webapp/stylesheets/teammates.css]: Add a space before`{`.

**Comment by samsontmr** [comment]
@damithc Currently, I have disabled the description check so the bot checks only the PR title. I'll enable the description check once I finish writing the code to construct the message is a modular way.

What I suspected happened was:
1. PR created -> payload sent with wrong title.
2. Bot comments.
3. Description modified -> payload sent with wrong title.
4. Bot comments.

**Comment by samsontmr** [comment]
@karthikaacharya The fist line of your PR description should be simply: `Fixes #<issue-number>`

**Comment by karthikaacharya** [comment]
Ok Sir.

**Comment by karthikaacharya** [comment]
Thank you for your support.

**Comment by junhaoyap** [comment]
@karthikaacharya I have set the PR status to ongoing, there are a few changes requested by @samsontmr and there are some conflicts to resolve

**Comment by karthikaacharya** [comment]
@junhaoyap I'll do it.

**Comment by karthikaacharya** [comment]
yes

**Comment by karthikaacharya** [comment]
@samsontmr  I used ctrl+shift+f in eclipse for formatting.

**Comment by samsontmr** [comment]
@karthikaacharya Well eclipse doesn't always get it right, and if you did configure your settings, I guess it didn't in this case. You'll just have to really on formatting it yourself in times like these.

**Comment by karthikaacharya** [comment]
@samsontmr ok sir.what should I do now?

**Comment by samsontmr** [comment]
1. Revert all the unnecessary modifications (this includes the lines where you modified the indentation).
1. Ensure your indentations are only four spaces wide.
1. Revert the line that you committed in `.settings/org.eclipse.jdt.core.prefs`. I believe you should not be committing that.
1. Fix the merge conflicts.

**Comment by karthikaacharya** [comment]
Ok.

**Comment by AnaghHegde** [comment]
@karthikaacharya Clean your repo. Its messed up.

**Comment by karthikaacharya** [comment]
I did not get any conflicts while rebasing 
fig3
I don't know why it is showing so much conflicts.Please see the attached image.
I got the required output also
fig5
can I send new pull request.

**Comment by karthikaacharya** [comment]
This is my first pull request(actually I am new to GitHub) please guide me through this.
I am gonna tell what I did from beginning...
1)I changed teammate.css and about.jsp, then I committed and I did Pull request  for these committed files, there was some indentation problem.
2)I fixed all these in my local machine and came to commit again then I came to know that teammate.css was not there in master branch of TEAMMATES(I think it got deleted).
   but teammate.css was there in my forked repository.Then I thought that my repository was not in sync with main repository. I merged all the commits that was there in teammates.git into my forked repo(I think this was the mistake).
3)Then I checked in my forked repo ,temmate.css was not there then I thought my repo was in sync with teammates.git(master branch).
4)then I made changes in teammtesCommon.css and about.jsp in my local machine and I committed it,after committing it it gave many  conflicts in other files which I not even touched
  .According to my knowledge conflicts occurs only when files I am working on is changed by others. so conflicts should occur in only about.jsp and teammatesCommon.css
  
  Now I don't know what do? if someone guide me it would be a great help.

**Comment by samsontmr** [comment]
Is your master branch up to date?

**Comment by karthikaacharya** [comment]
yes.

**Comment by karthikaacharya** [comment]
@Samson Tan when I compare my master branch with TEMMATES it shows no
difference.

**Comment by samsontmr** [comment]
Since it's updated maybe you can try what I suggested?

**Comment by karthikaacharya** [comment]
Ok I'll do it.

**Comment by karthikaacharya** [comment]
I will create once again ,since there were so many conflicts I am redoing it everything.

**Pull request closed without merging**